frontend/packages/dev-console/src/components/dropdown/__tests__/ResourceDropdown.spec.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import { mount } from 'enzyme';
```

Comment:
Use ```shallow``` instead of ```mount```. ```mount``` render the children as well which is not needed. If needed you can use shallow again on the children component. And ```shallow``` is faster then ```mount``` as it not render the children component.

Reply:
Changed to `shallow` other than last test. `mount` is required for the last test case.

Reply:
You can use shallow for the last test as well. I suggested the changes.

Reply:
done